src/components/Picklist/pageObject/index.js

```diff
@@ -136,28 +136,20 @@ class PagePicklist {
     }
 
     /**
-     * Returns true if the bounds of PicklistOption page object of the element in item position.
-     * is contained within Picklist menu bounds rect.
-     * @param {number} optionIndex - The base 0 index of the PicklistOption.
+     * Returns the boundaries of Picklist dropdown menu.
+     * @method
+     * @returns {object}
      */
-    isOptionVisibleWithinMenuBounds(optionIndex) {
-        const isPositionWithinBoundsRect = (position, boundsRect) => {
-            const { x, y } = position;
-            const { l, t, r, b } = boundsRect;
-            return x >= l && y >= t && x <= r && y <= b;
-        };
-
+    getDropdownMenuBoundsRect() {
```

Comment:
Remove this comment since it is for documentation and this method is private.
Then to make this like a private method we need to make this with a Symbol
oustide the class:
```const privateGetMenuBoundsRect = Symbol('privateGetMenuBoundsRect');```
and then when declare the method:
```
[privateGetMenuBoundsRect]() {

}
```
then use it like this:
```return new PagePicklistOption(option, this[privateGetMenuBoundsRect]());```